readme.md

```diff
@@ -633,6 +633,7 @@
 - [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.
 - [Connectivity Data and Reports](https://github.com/stevesong/awesome-connectivity-info#readme) - Better understand who has access to telecommunication and internet infrastructure and on what terms.
 - [Stacks](https://github.com/stackshareio/awesome-stacks#readme) - Tech stacks for building different apps and features.
+- [Morphological Profiling](https://github.com/cytodata/awesome-cytodata) - Resources for computational biologists to acquire and disseminate knowledge about image-based profiling of biological phenotypes.
```

Comment:
```suggestion
- [Morphological Profiling](https://github.com/cytodata/awesome-cytodata#readme) - Resources for computational biologists to acquire and disseminate knowledge about image-based profiling of biological phenotypes.
```

And would be good to try to exclude "Resources for" from the description. Every list has resources for something, and the descriptions would not be very good if every list description had that prefix.

Maybe: `Image-based profiling of biological phenotypes for computational biologists.` ?